server/src/users/entities/program.entity.ts

```diff
@@ -0,0 +1,14 @@
+import { Column, Entity, ManyToMany, PrimaryGeneratedColumn } from 'typeorm';
+import { User } from './user.entity';
+
+@Entity('programs')
+export class Program {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @ManyToMany((type) => User, (user) => user.subscriptions)
+  subscribers: User[];
```

Comment:
maybe add a description for the program?

Reply:
Done👍